index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Anja Schmidt's Portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div class="mobile_container">
+      <!-- Nav Bar -->
+      <header>
+        <ul>
+          <li><a href="#" rel="noopener noreferrer">Welcome</a></li>
+          <li><img class="menu" src="images/menu.png" alt="Menu" /></li>
+        </ul>
+      </header>
+
+      <!-- Header -->
+
+      <section id="main">
+        <section class="header_container">
+          <h1 class="heading heading1">Hey there. I’m Anja.</h1>
+          <p class="heading heading2">I’m a software developer</p>
```

Comment:
- [ ] Good job removing the double `<h1>` as the previous CR mentioned. 👍 however according to the requirements the subtitle should still be a header tag `hx (examples: <h2>,<h3>, etc...)`